.github/workflows/cmake.yml

```diff
@@ -153,6 +157,27 @@ jobs:
             -DGRN_WITH_APACHE_ARROW=ON \
             -DGRN_WITH_BLOSC=auto \
             -DGRN_WITH_MRUBY=${GRN_WITH_MRUBY:-ON}
+      - name: CMake (with UBSAN)
+        if: matrix.ubsan == 'ON'
+        run: |
+          ccache --show-stats --verbose --version || :
+          CLANG_VER=$(clang -dumpversion | cut -d. -f1)
+          CLANG_LIBRARY_PATH="/usr/lib/clang/${CLANG_VER}/lib/linux"
+          ls -la /usr/lib/clang/${CLANG_VER}/lib/linux
+          echo "LD_LIBRARY_PATH=${CLANG_LIBRARY_PATH}:${LD_LIBRARY_PATH}" >> ${GITHUB_ENV}
+          cmake \
+            -B ../groonga.build \
+            -S . \
+            -GNinja \
+            -DCMAKE_INSTALL_PREFIX=$PWD/install \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DGRN_ALLOW_WARNING=OFF \
+            -DGRN_WITH_APACHE_ARROW=ON \
+            -DGRN_WITH_BLOSC=auto \
+            -DGRN_WITH_MRUBY=${GRN_WITH_MRUBY:-ON} \
+            -DCMAKE_C_FLAGS="-fsanitize=undefined -fno-sanitize=alignment" \
+            -DCMAKE_CXX_FLAGS="-fsanitize=undefined -fno-sanitize=alignment" \
+            -DCMAKE_EXE_LINKER_FLAGS="-L${CLANG_LIBRARY_PATH} -lclang_rt.ubsan_standalone-x86_64"
```

Comment:
Should we enable UBSAN using CMake options like `-DWITH_UBSAN=ON`?

Reply:
Yes. We should not use raw `CMAKE_{C,CXX}_FLAGS`  and so on for it.

Reply:
Sure. I will address it.

Reply:
fix: [64d18ac](https://github.com/groonga/groonga/pull/2277/commits/64d18acb042510b543bf627cc98e7397120f97fc) Thanks. I've added `GRN_WITH_UBSAN` option here.